internal/kymacustomresource/kyma_cr_updater.go

```diff
@@ -17,6 +17,7 @@ const (
 	namespace               = "kcp-system"
 	subaccountIdLabelKey    = "kyma-project.io/subaccount-id"
 	subaccountIdLabelFormat = "kyma-project.io/subaccount-id=%s"
+	k8sRequestInterval      = 5 * time.Second
```

Comment:
Can you create a test that sets shorter timeout and asserts that function execution fails with timeout?

Reply:
Too much hassle.